src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java

```diff
@@ -967,12 +967,28 @@ public boolean checkRecipeGeneric(
             return false;
         }
 
+        float batchMultiplier = 1.0f;
+        if (mUseMultiparallelMode) {
+            int extraParallelRecipes = 0;
+            for (; extraParallelRecipes + parallelRecipes < aMaxParallelRecipes * 128; extraParallelRecipes++) {
```

Comment:
Also imo this should be done in the main parallel loop instead of being its own thing, since this loop ignores energy while the other respects if the machine has enough energy for so many parallels

Reply:
Nvmd blue reminded me thats now how batch mode works 💀

Reply:
yeah, the increase in recipe time already covers the power cost